hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -920,6 +920,14 @@
   </description>
 </property>
 
+<property>
+  <name>io.erasurecode.codec.native.enabled</name>
+  <value>true</value>
+  <description>
+    Used to decide whether to enable native codec.
```

Comment:
This needs more explanation. In what cases is it a good idea to disable ISA-L support?